app/controllers/api/v2/notifications_controller.rb

```diff
@@ -0,0 +1,42 @@
+module API
+  module V2
+    class NotificationsController < API::V2::ApplicationController
+      before_action :build_provider
+      deserializable_resource :user_notification,
+                              class: API::V2::DeserializableNotification
+      def create
```

Comment:
It was suggested in the ticket that we go for a single endpoint to create and update notifications but inclined to thing we should go for a separate update endpoint unless people strongly disagree?

Reply:
I agree with you about a separate update endpoint but does it have to be another PR?

Reply:
+1 for a separate endpoint. Mostly because having a single endpoint for update and create isn't a pattern we have so it will be more consistent with the rest of the API. I'm easy on whether you do it here or another PR.